src/create-config.ts

```diff
@@ -0,0 +1,317 @@
+        const sponsor = chain.sponsor;
+        const beaconUpdateSubscriptionId = ethers.utils.keccak256(
+          ethers.utils.defaultAbiCoder.encode(
+            ['uint256', 'address', 'bytes32', 'bytes', 'bytes', 'address', 'address', 'address', 'bytes4'],
+            [
+              chainId,
+              airnodeAddress,
+              templateId,
+              parameters,
+              encodedBeaconUpdateSubscriptionConditions,
+              airnodeAddress,
+              sponsor,
+              DapiServerAddress,
+              DapiServerInteface.getSighash('fulfillPspBeaconUpdate'),
+            ]
+          )
+        );
+        return {
+          [beaconUpdateSubscriptionId]: {
+            chainId: `${chainId}`,
+            parameters,
+            airnodeAddress,
+            templateId,
+            conditions: encodedBeaconUpdateSubscriptionConditions,
+            relayer: airnodeAddress,
+            sponsor,
+            requester: DapiServerAddress,
+            fulfillFunctionId: DapiServerInteface.getSighash('fulfillPspBeaconUpdate'),
+          },
+        };
+      });
+    })
+    .reduce((subscriptionsObject, subscription) => ({ ...subscriptionsObject, ...subscription }), {});
+
+  const airkeeperTriggers = {
+    rrpBeaconServerKeeperJobs: [],
+    protoPsp: Object.keys(airkeeperSubscriptions),
+  };
+
+  const airkeeperTemplates = Object.values(apiData.templates).reduce(
+    (templateObj, template) => ({
+      ...templateObj,
+      [template.templateId]: {
+        endpointId: template.endpointId,
+        templateParameters: template.parameters,
+      },
+    }),
+    {}
+  );
+
+  const airkeeperEndpointArray = await Promise.all(
+    Object.values(apiData.ois).flatMap((ois) => {
+      return ois.endpoints.map(async (endpoint) => ({
+        [await deriveEndpointId(ois.title, endpoint.name)]: { endpointName: endpoint.name, oisTitle: ois.title },
+      }));
+    })
+  );
+
+  const AirkeeperEndpoints = airkeeperEndpointArray.reduce(
+    (endpointsObject, endpoint) => ({ ...endpointsObject, ...endpoint }),
+    {}
+  );
+
+  const AirkeeperConfig: AirkeeperConfig = {
+    airnodeAddress: apiData.apiMetadata.airnode,
+    airnodeXpub: apiData.apiMetadata.xpub,
+    chains: airkeeperChains,
+    triggers: airkeeperTriggers,
+    subscriptions: airkeeperSubscriptions,
+    templates: airkeeperTemplates,
+    endpoints: AirkeeperEndpoints,
+  };
+
+  // Get the current date
+  const date = new Date().toISOString().split('T')[0];
+
+  //// Create the deployment directory ////
+  const updatedOpsData: OperationsRepository = {
+    ...operationsRepository,
+    apis: {
+      ...operationsRepository.apis,
+      [response.apiName]: {
+        ...operationsRepository.apis[response.apiName],
+        deployments: {
+          ...operationsRepository.apis[response.apiName].deployments,
+          [date]: {
+            config: Config,
+            airkeeper: AirkeeperConfig,
```

Comment:
If you rename AirkeeperConfig's declaration to airkeeper you won't have to re-assign here.